src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1487,7 +1488,9 @@ export class ResultsViewPageStore {
 
                 if (
                     profile.molecularAlterationType ===
-                    AlterationTypeConstants.MUTATION_EXTENDED
+                        AlterationTypeConstants.MUTATION_EXTENDED ||
+                    profile.molecularAlterationType ===
+                        AlterationTypeConstants.MUTATION_UNCALLED
```

Comment:
@dippindots for the `MUTATION_UNCALLED` we should prolly have another `if` category. It should be removed and not show up anywhere on resultsview page.

Reply:
Thanks @inodb, I will update this pr!